mint-auditor/migrations/2022-05-24-214504_create_initial/up.sql

```diff
@@ -123,6 +123,19 @@ CREATE TABLE gnosis_safe_withdrawals (
 CREATE INDEX idx__gnosis_safe_withdrawals__eth_block_number ON gnosis_safe_withdrawals(eth_block_number);
 CREATE INDEX idx__gnosis_safe_withdrawals__mc_tx_out_public_key_hex ON gnosis_safe_withdrawals(mc_tx_out_public_key_hex);
 
+-- Mapping between MintTxs and GnosisSafeDeposits that match each other.
+-- This essentially is the audit log that shows which mints/deposits were a match.
+-- A match means that the nonce, token information (MC token_id and Ethereum contract address) and the amount all matched.
+-- If a mint or deposit are not referenced by this table that means something questionable happened.
+CREATE TABLE audited_mints (
+    id INTEGER PRIMARY KEY,
+    mint_tx_id INTEGER NOT NULL,
+    gnosis_safe_deposit_id INTEGER NOT NULL,
+    -- Constraints
+    FOREIGN KEY (mint_tx_id) REFERENCES mint_txs(id),
+    FOREIGN KEY (gnosis_safe_deposit_id) REFERENCES gnosis_safe_deposits(id)
```

Comment:
I'll remove the comment about keeping Diesel happy. I don't think we should reference tables via these fields instead of the id fields (elaborated why on the other comment)